<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifierDialog</class>
 <widget class="QDialog" name="ModifierDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>781</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(70, 126, 171);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>220</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/Desktop/img/5973800-removebg-preview.png</normaloff>:/Desktop/img/5973800-removebg-preview.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>500</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/Desktop/img/1623971-removebg-preview.png</normaloff>:/Desktop/img/1623971-removebg-preview.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>410</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/Desktop/img/9321345-removebg-preview.png</normaloff>:/Desktop/img/9321345-removebg-preview.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>590</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/Desktop/img/4986021-removebg-preview.png</normaloff>:/Desktop/img/4986021-removebg-preview.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none; /* Supprime le fond */
qproperty-iconSize: 70px 70px; /* Augmente la taille de l'icône */
border: none; /* Supprime la bordure si nécessaire */

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/Desktop/img/logo_final-removebg-preview.ico</normaloff>:/Desktop/img/logo_final-removebg-preview.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>320</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/Desktop/img/3079026-removebg-preview.png</normaloff>:/Desktop/img/3079026-removebg-preview.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>751</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 25px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent ;
background-color: transparent;
qproperty-iconSize: 30px 30px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : none ;
font: 700 18pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Modifier l'employé </string>
    </property>
   </widget>
   <widget class="QPushButton" name="liste">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
font: 700 9pt &quot;Segoe UI&quot;;
border: 2px solid black ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
qproperty-iconSize: 20px 20px; /* Ajuste la taille de l'icône */
border-radius: 20px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
    </property>
    <property name="text">
     <string>Liste</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/Desktop/img/570170-removebg-preview[1].png</normaloff>:/Desktop/img/570170-removebg-preview[1].png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="stat">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
font: 700 9pt &quot;Segoe UI&quot;;
border: 2px solid black ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
qproperty-iconSize: 20px 20px; /* Ajuste la taille de l'icône */
border-radius: 20px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
    </property>
    <property name="text">
     <string>Statistiques</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/Desktop/img/1170667-removebg-preview.png</normaloff>:/Desktop/img/1170667-removebg-preview.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="ajout">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
font: 700 9pt &quot;Segoe UI&quot;;
border: 2px solid black ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
qproperty-iconSize: 30px 30px; /* Ajuste la taille de l'icône */
border-radius: 20px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
    </property>
    <property name="text">
     <string>Ajouter
 employe</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/Desktop/img/18926930-conception-d-icone-de-formulaire-gratuit-vectoriel-removebg-preview[1].png</normaloff>:/Desktop/img/18926930-conception-d-icone-de-formulaire-gratuit-vectoriel-removebg-preview[1].png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>781</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 186, 196);
border: 2px solid black ; /* Ajoute une bordure */
border-radius: 25px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
     </property>
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
     </property>
     <property name="text">
      <string>Poste</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenom">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="telephone">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="email">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:  transparent ;
font: 700 12pt &quot;Arial Rounded MT&quot;;
background : none ;</string>
     </property>
     <property name="text">
      <string>Téléphone
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
     </property>
     <property name="text">
      <string>Statut</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
     </property>
     <property name="text">
      <string>Date_Embauche</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
background : none ;</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="poste">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>280</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="statut">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_embauche">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
     </property>
     <property name="text">
      <string>Sexe</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
     </property>
     <property name="text">
      <string>Salaire</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="salaire">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sexe">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Homme</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Femme</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>241</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(176, 186, 196);
border : none ;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 13pt &quot;Arial Rounded MT&quot;;
</string>
      </property>
      <property name="text">
       <string>Photo de profil</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pdp1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>131</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_modifierPhoto">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>250</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(70, 126, 171);
font: 900 10pt &quot;Segoe UI&quot;;
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 20px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
     </property>
     <property name="text">
      <string>Modifier Photo de Profil </string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::WindowNew"/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_enregistrer">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>310</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 3px solid black;
background-color: rgb(132, 194, 49);
border-radius: 15px;
font: 700 10pt &quot;Arial Rounded MT&quot;;</string>
     </property>
     <property name="text">
      <string>Enregistrer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pdp2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">qproperty-iconSize: 70px 70px; /* Ajuste la taille de l'icône */
border : transparent ;
background : none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Downloads/istockphoto-1474007937-170667a-removebg-preview.png</normaloff>:/Downloads/istockphoto-1474007937-170667a-removebg-preview.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>120</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/Desktop/img/employe.png</normaloff>:/Desktop/img/employe.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
